README.md

```diff
@@ -1,3 +1,20 @@
-# HWApp
+# HWApp [![CI Tests](https://github.com/georgiybykov/hw_app/actions/workflows/elixir.yml/badge.svg)](https://github.com/georgiybykov/hw_app/actions)
 
-Hello World application as tutrial one with test and CI.
+Hello World application as tutrial one with:
+ - linter;
+ - type checking;
+ - tests;
+ - configured GitHub Actions.
+ 
+##### Dependecies:
```

Comment:
I doubt this is actually needed to be exposed in `README` :)

Reply:
Of course! :D